internal/cli/analytics/reporter.go

```diff
@@ -0,0 +1,39 @@
+package analytics
+
+import "github.com/kubeshop/botkube/internal/cli"
+
+const (
+	defaultCliVersion = "v9.99.9-dev"
+)
+
+var (
+	// APIKey contains the API key for external analytics service. It is set during application build.
+	APIKey string = ""
+)
+
+// Reporter defines behavior for reporting analytics.
+type Reporter interface {
+	ReportCommand(cmd string) error
+	ReportError(err error, cmd string) error
+	Close()
+}
+
+// NewReporter creates a new Reporter instance.
+func GetReporter() Reporter {
```

Comment:
Can we print just a single line which reporter is used when the global verebose flag (`-v/--verbose`) is used? That would be helpful in debugging scenario.